ports/mdl/portfile.cmake

```
@@ -0,0 +1,125 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+
+# Clang 7 build tool
+
+set(LLVM_VERSION 7.0.0)
+set(LLVM_BASE_URL "https://releases.llvm.org/${LLVM_VERSION}")
+
+if(VCPKG_HOST_IS_WINDOWS)
+    set(LLVM_FILENAME  "LLVM-${LLVM_VERSION}-win64.exe")
+    set(LLVM_HASH      c2b1342469275279f833fdc1e17ba5a9f99021306d6ab3d7209822a01d690767739eebf92fd9f23a44de5c5d00260fed50d5262b23a8eccac55b8ae901e2815c)
+elseif(VCPKG_HOST_IS_LINUX)
+    set(LLVM_FILENAME  "clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-ubuntu-16.04.tar.xz")
+    set(LLVM_HASH      fb3dc588137426dc28a20ef5e34e9341b18114f03bf7d83fafbb301efbfd801bba08615b804817c80252e366de9d2f8efbef034e53a1b885b34c86c2fbbf9c28)
+elseif(VCPKG_HOST_IS_FREEBSD)
+    set(LLVM_FILENAME  "clang+llvm-${LLVM_VERSION}-amd64-unknown-freebsd11.tar.xz")
+    set(LLVM_HASH      d501484c38cfced196128866a19f7fef1e0b5d609ea050d085b7deab04ac8cc2bbf74b3cfe6cd90d8ea17a1d9cfca028a6c933f0736153ba48785ddc8646574f)
+elseif(VCPKG_HOST_IS_OSX)
+    set(LLVM_FILENAME  "clang+llvm-${LLVM_VERSION}-x86_64-apple-darwin.tar.xz")
+    set(LLVM_HASH      c5ca6a7756e0cecdf78d4d0c522fe7e803d4b1b2049cb502a034fe8f5ca30fcbf0e738ebfbc89c87de8adcd90ea64f637eb82e9130bb846b43b91f67dfa4b916)
+else()
+    message(FATAL_ERROR "Pre-built binaries for Clang 7 not available, aborting install (platform: ${VCPKG_CMAKE_SYSTEM_NAME}, architecture: ${VCPKG_HOST_ARCHITECTURE}).")

```

```suggestion
    message(FATAL_ERROR "Pre-built binaries for Clang 7 not available, aborting install (platform: ${VCPKG_CMAKE_SYSTEM_NAME}).")
```

Maybe we can wait for #25529 to define `VCPKG_HOST_TRIPLET`

---

I am fine to wait, but given the lengthy PR why not doing it as a follow-up?